src/client-services/github.client-service.ts

```diff
@@ -1,33 +1,38 @@
-import axios, { AxiosResponse, AxiosInstance } from 'axios';
+import axios, { AxiosResponse, AxiosInstance } from 'axios'
 import getConfig from 'next/config'
 
 function axiosConfig(): AxiosInstance {
   const { serverRuntimeConfig } = getConfig()
-  const accessToken: string | undefined = serverRuntimeConfig.githubApiToken;
+  const accessToken: string | undefined = serverRuntimeConfig.githubApiToken
 
   if (!accessToken) {
-    throw new Error('GitHub access token is not provided.');
+    throw new Error('GitHub access token is not provided.')
   }
 
   const axiosInstance = axios.create({
     baseURL: 'https://api.github.com',
     headers: {
-      'Authorization': `token ${accessToken}`,
+      Authorization: `token ${accessToken}`,
       'Content-Type': 'application/json',
     },
-  });
+  })
 
-  return axiosInstance;
+  return axiosInstance
 }
 
 export async function getReposList(): Promise<AxiosResponse> {
-  const response: AxiosResponse = await axiosConfig().get('/orgs/OpenBCca/repos');
-  return response;
+  const response: AxiosResponse = await axiosConfig().get(
+    '/orgs/OpenBCca/repos'
+  )
+  return response
 }
 
-export async function getRepoInfo(repo_name: string, para: string = ''): Promise<AxiosResponse> {
-  const response = await axiosConfig().get(`/repos/OpenBCca/${repo_name}${para}`);
-  return response;
+export async function getRepoInfo(
+  repo_name: string,
+  para: string
```

Comment:
@SamHuo213 I got this error
```
Error: Type string trivially inferred from a string literal, remove type annotation.  @typescript-eslint/no-inferrable-types
```
for `para: string = ''`, so I changed it to `para: string` here

Reply:
We need to update the lint configurations.

Also, @PooriaT needs to fix the build.

At this time can, you can ignore it I guess.